_sass/template/partials/_pdf-figures.scss

```diff
@@ -209,27 +209,39 @@ $pdf-figures: true !default;
     // Get the proportions we need
     $rotator-content-width: $page-width - $margin-inside - $margin-outside;
     $rotator-content-height: $page-height - $margin-top - $margin-bottom;
-    $rotator-content-wh-ratio: (1 - $rotator-content-width/$rotator-content-height) * 100%;
+    $rotator-content-sidebar-width: $sidebar-content-width + $sidebar-gap-width;
 
     .figure-rotator-wrapper {
         // First we float the wrapper to the top of the page,
         // and set its height and width to match the content area.
         float: top;
         height: $rotator-content-height;
         width: $rotator-content-width;
+        break-before: verso;
+        break-after: page;
+
+        // When extending into the sidebar, add its width
+        &.pdf-wide {
+            float: sidenote;
+            width: $rotator-content-width + $rotator-content-sidebar-width;
+
+            .figure-rotator {
+                height: $rotator-content-width + $rotator-content-sidebar-width;
+            }
+        }
 
         .figure-rotator {
             // First we reverse the rotating div's proportions.
             height: $rotator-content-width;
-            width: $rotator-content-height;
+            width: $rotator-content-height; // may be overridden by greater specifity above
             // Then we get this div into its new position:
-            // 1. We choose to rotate from the bottom-left corner.
-            transform-origin: bottom left;
+            // 1. We choose to rotate from the top-left corner.
+            transform-origin: top left;
             // 2. Rotate the div anti-clockwise 90%.
-            // 3. Shift the div right by its own (now) width.
-            // 4. Shift the div down by the proportion of the content area's width:height.
+            // 4. Shift the div down by the content area's height,
+            //    rememebering that, rotated, 'down' is x not y.
             // Now it's exactly in position.
-            transform: rotate(-90deg) translatey(100%) translatex(-($rotator-content-wh-ratio));
+            transform: rotate(-90deg) translatex(-($rotator-content-height));
```

Comment:
I believe the [convention](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) is to use uppercase X in 'translateX', to make it easier to read